assets/course-theme/blocks/lesson-blocks/course-title/course-title-edit.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps } from '@wordpress/block-editor';
+
+export const CourseTitleEdit = () => {
+	const blockProps = useBlockProps();
+
+	// Todo: Add the heading level selection, similar to this https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/site-title/edit/index.js#L98-L103.
```

Comment:
I think it is better to create an issue for that instead of leaving as a comment.